ultra/tests/scenarios/maps/easy/map-flow-0/map.net.xml

```diff
@@ -1,135 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are these map files not needed anymore?

Reply:
Every time I run the tests, these files seem to be deleted in git. I do not know why? But yeah those are only testing

Reply:
Oh I see. If this pull request is merged to develop though, will these files no longer be in the develop branch? I think they are needed to run the tests.